tests/test_interface.rb

```diff
@@ -629,7 +629,7 @@ def test_mtu_invalid_loopback
   end
 
   def test_speed
-    skip_legacy_defect?('7.0.3.I5.2', 'CSCvd41419')
+    skip_legacy_defect?('7.0.3.I5|7.0.3.F3', 'CSCvd41419')
```

Comment:
Is this defect still present in `7.0.3.I5.3`?

Reply:
yes